src/components/pages/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const Form = (title, author) => (
+  <>
+    <div className="form-container">
+      <h2 className="form-header">Add New Book</h2>
+      <form action="POST" id="form" className="form">
```

Comment:
There seems to be something weird about this component. From a standard react perspective, all components take in a `props` param which is an object of other param but in this case the `title` and `author` params where used directly like a normal function. Can you please review this again?, also prop-types is missing here

Reply:
Resolved